.github/workflows/generate_translations.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           python3 -m pip install pip==20.0.2
           python3 -m pip install setuptools==50.3.0
           python3 -m pip install wheel==0.35.1
-          git clone https://asadali145:${ASAD_PAT}@github.com/OmnipreneurshipAcademy/edx-msp-assessment-xblock.git@${ADG_XBLOCKS_BRANCH}#egg=msp-assessment-xblock
+          git clone https://${ASAD_PAT}@github.com/OmnipreneurshipAcademy/edx-msp-assessment-xblock.git@${ADG_XBLOCKS_BRANCH}#egg=msp-assessment-xblock
```

Comment:
Why there is ASAD_PAT here?

Reply:
Please ignore, Testing a few things as requirements are not being installed.